app/models/submit_source.rb

```diff
@@ -0,0 +1,15 @@
+class SubmitSource < MailForm::Base
+  attribute :url,       :validate => true
+  attribute :email,     :validate => /\A([\w\.%\+\-]+)@([\w\-]+\.)+([\w]{2,})\z/i
```

Comment:
Line is too long. [81/80]<br>Unnecessary spacing detected.<br>Use the new Ruby 1.9 hash syntax.